src/views/CampusSafety/views/LostAndFound/views/ReportFound/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, Card, CardContent, CardHeader, Grid, Typography } from '@mui/material';
+import { LocalLibrary, LocalPolice } from '@mui/icons-material';
+import { useNavigate } from 'react-router';
+import Header from 'views/CampusSafety/components/Header';
+
+const ReportFound = () => {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <Header />
+      <Card>
+        <CardHeader title={'Report a Found Item'} className="gc360_header" />
+        <CardContent>
+          <Typography>If you've found an item somewhere on campus:</Typography>
+          <ul>
```

Comment:
This is just my opinion, but I think this looks a little weird as a bulleted list.

Reply:
Yeah, if you have ideas for how to design this page better, let me know.  I kinda agree, but I also don't really know what would work better.  I wrote this page in about 15 minutes, so it's kinda just thrown together.

Reply:
I'm not exactly sure either. Maybe we can all figure out how it should look on Tuesday? I would say you could merge it for now.